packages/@uppy/companion/src/server/logger.js

```diff
@@ -1,5 +1,19 @@
 const chalk = require('chalk')
 
+const valuesToMask = []
+/**
+ * Adds a list of strings that should be masked by the logger.
+ * This function can only be called once through out the life of the server.
+ * @param {Array} maskables a list of strings to be masked
+ */
+exports.addMaskables = (maskables) => {
```

Comment:
If it can only be called once, perhaps a more appropriate name is something like `setMaskables`? `addX` sounds like you can add many `X`es by calling it many times

Reply:
changed